package.json

```diff
@@ -9,6 +9,8 @@
     "start": "npm run build && node ./bin/www",
     "fakeApi": "json-server -p 4000 db.json",
     "test": "npm run lint && mocha tests/unit/**/*.ts --require ts-node/register",
+    "coverage": "nyc npm test",
+    "check-coverage": "npm run test-coverage && nyc check-coverage --lines 70",
```

Comment:
can you update npm run test-coverage to npm run coverage. Update TODO on README to detail two checks and I'll merge this.